frontend/src/pages/Compare.test.tsx

```diff
@@ -212,9 +345,7 @@ describe('Switch between v1 and v2 Run Comparison pages', () => {
     getRunSpy.mockImplementation((id: string) => runs.find(r => r.run!.id === id));
 
     // v2 feature is turn off.
-    jest.spyOn(features, 'isFeatureEnabled').mockImplementation(_ => {
-      return false;
-    });
+    jest.spyOn(features, 'isFeatureEnabled').mockImplementation(_ => false);
```

Comment:
NIT: You can use `mockReturnValue(false)` instead of `mockImplementation()`

https://jestjs.io/docs/mock-function-api#mockfnmockreturnvaluevalue

Reply:
Ah that's much simpler, thank you! Done ✅